PLATEXP-1702: Modified add-cpu-to-server-or-hypervisor (#1417)
rackerlabs/support-how-to: content/how-to/dedicated-hosting/add-cpu-to-server-or-hypervisor/index.md

**Step 3.** Under **Common Request**, click **See All Templates**. The **See All Templates** displays list of all templates.

   <img width="958" alt="Displays the list of available templates" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor6.png">

**Step 4.** From the **Common Request** menu, select **Add CPU to Server or Hypervisor**.

  <img width="249" alt="Select add CPU to server or hypervisor from common request menu" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor7.png">

**Step 5.** In the **Issue Details** section, select a device from the drop-down menu and
   enter the number of CPUs that you want to add.

  <img width="524" alt="Select the required device from drop-down menu" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor8.png">

**Step 6.** In the **Preferred Maintenance Schedule** section, you can choose to have the
   work done at any time by selecting the **Anytime** checkbox. Fill out the required fields such as contact name, phone number and select the appropriate radio button as shown in following image. 

>    <img width="275" alt="Checkbox Anytime is tick so that you don't have to decide day and time" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor9.png">

Reviewer: ```suggestion
   <img width="275" alt="Checkbox Anytime is selected so that you don't have to decide day and time" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor9.png">
```

>    <img width="275" alt="Checkbox Anytime is tick so that you don't have to decide day and time" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor9.png">

Reviewer: ```suggestion
   <img width="275" alt="Checkbox Anytime is selected so that you don't have to decide day and time" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor9.png">
```

>    If checkbox for **Anytime** is deselect, then set the preferred maintainance day and time.

Reviewer: ```suggestion
   If checkbox for **Anytime** is deselected, then set the preferred maintenance day and time.
```

>    fill out the required maintainance fields to have the work completed in a scheduled maintenance window.

Reviewer: ```suggestion
   Fill out the required maintenance fields to have the work completed in a scheduled maintenance window.
```

> 1. Log in to the [MyRackspace portal](https://login.rackspace.com/login) with

Author: Updated

   your username and password.

   <img width="409" alt="hypervisor4" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor4.png">

2. In the top navigation bar, choose **Support** > **Support Center**.

   <img width="941" alt="hypervisor5" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor5.png">

3. Under **Common Request**, click **See All Templates**. The **See All Templates** displays list of all templates.

   <img width="958" alt="hypervisor6" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor6.png">

> 5. From the **Common Request** menu, Select **Add CPU to Server or Hypervisor**.

Author: Changed

  <img width="249" alt="hypervisor7" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor7.png">

6. In the **Issue Details** section, select a device from the drop-down menu and
   enter the number of CPUs that you want to add.

  <img width="524" alt="hypervisor8" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor8.png">

> 7. In the **Preferred Maintenance Schedule** section, you can choose to have the

Author: Change as suggested.

   work done at any time by selecting the **Anytime** checkbox. Otherwise, fill
   out the remaining maintenance details to have the work completed in a
   scheduled maintenance window.

8. Click **Submit**.

>    <img width="195" alt="hypervisor9" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor9.png">

Author: Done
